app/models/manageiq/providers/openstack/inventory/parser/cloud_manager.rb

```diff
@@ -173,6 +173,8 @@ def orchestration_stack_resources(stack, stack_inventory_object)
     raw_resources.reject! { |r| r.physical_resource_id.nil? }
```

Comment:
There's already a check on this line for nil, just like in the old refresher. I'm guessing it should be changed to a check for blank?

Reply:
hum, you might be right

Reply:
yeah, the build method checks blank? https://github.com/Ladas/manageiq/blob/17c884ab58ea6e7f0aad11dd87982adf24409c17/app/models/manager_refresh/inventory_collection.rb#L754